### ISSUE TITLE
App Recommendations

### ISSUE THREAD

**Issue by Sepero** [39aldo39/DecSyncCC#69]
I've been wanting to use Decsync for nearly a year, but I've never had the time to figure out the details of whatever it takes to set it up. I already run Syncthing on my mobile device, so I'm ready to sync, but Decsync doesn't connect to my Contacts and Calendar.

It appears I need to install some extra Calendar or Contacts apps that will connect with Decsync? I really don't care what I have to do, but I need the instructions on how to do it. We all have busy lives. Please, in the app, provide some instructions on how to get it going (or a link from the app to a webpage with instructions). Also, if I need to install new apps, then please recommend some.

Thank you

**Comment by 39aldo39** [comment]
I agree that it can be clearer. However, I am also working on an import function directly from DecSync CC. I prefer that more convenient solution which hopefully is included in the next release.
